src/directory/directory.mjs

```diff
@@ -2,210 +2,1809 @@
  * @type {import('./directory').PageNode}
  */
 export const directory = {
```

Comment:
used a script to fill in directory.mjs, this does not maintain the order that was previously established and instead just recursively iterated over the files that existed.  Will probably want to regenerate with ordering in mind.